sdk/search/Azure.Search.Documents/tests/Spatial/SpatialProxyFactoryTests.cs

```diff
@@ -13,125 +13,125 @@ public class SpatialProxyFactoryTests
 
             Assert.AreSame(line, proxy.Value);
-            Assert.AreEqual(5, line.Points.Count);
-            Assert.AreEqual(line.Points.Count, proxy.Points.Count);
+            Assert.AreEqual(5, line.Coordinates.Count);
+            Assert.AreEqual(line.Coordinates.Count, proxy.Coordinates.Count);
 
-            for (int i = 0; i < line.Points.Count; i++)
+            for (int i = 0; i < line.Coordinates.Count; i++)
             {
-                Assert.AreEqual(line.Points[i].Latitude, proxy.Points[i].Latitude);
-                Assert.AreEqual(line.Points[i].Longitude, proxy.Points[i].Longitude);
+                Assert.AreEqual(line.Coordinates[i].Latitude, proxy.Coordinates[i].Latitude);
+                Assert.AreEqual(line.Coordinates[i].Longitude, proxy.Coordinates[i].Longitude);
             }
         }
 
         [TestCaseSource(nameof(CreateFailsData))]
         public void CreateFails(object value)
         {
-            Assert.IsFalse(SpatialProxyFactory.TryCreate(value, out GeographyProxy proxy));
+            Assert.IsFalse(SpatialProxyFactory.TryCreate(value, out GeoObjectProxy proxy));
             Assert.IsNull(proxy);
         }
 
         [TestCase(null, false)]
         [TestCase(typeof(object), false)]
         [TestCase(typeof(int), false)]
-        [TestCase(typeof(GeographyPoint), true)]
-        [TestCase(typeof(GeometryPoint), false)]
-        [TestCase(typeof(GeographyPosition), false)]
-        [TestCase(typeof(GeometryPosition), false)]
-        [TestCase(typeof(GeographyPolygon), false)]
-        [TestCase(typeof(GeometryPolygon), false)]
-        [TestCase(typeof(GeographyLineString), false)]
-        [TestCase(typeof(GeometryLineString), false)]
+        [TestCase(typeof(GeoPoint), true)]
+        [TestCase(typeof(GeoLinearRing), false)]
+        [TestCase(typeof(GeoLineString), false)]
+        [TestCase(typeof(GeoPolygon), false)]
+        [TestCase(typeof(GeoPosition), false)]
         public void IsSupportedPoint(Type type, bool expected) =>
             Assert.AreEqual(expected, SpatialProxyFactory.IsSupportedPoint(type));
 
         private static IEnumerable CreateFailsData => new[]
         {
             new TestCaseData(new object()),
             new TestCaseData(1),
-            new TestCaseData(GeometryFactory.Point(1.0, 2.0)),
-            new TestCaseData(new GeometryPosition(1.0, 2.0)),
-            new TestCaseData(GeographyFactory
-                .Polygon()
-                .Ring(0.0, 0.0)
-                .LineTo(1.0, 0.0)
-                .LineTo(1.0, 1.0)
-                .LineTo(0.0, 1.0)
-                .LineTo(0.0, 0.0)),
-            new TestCaseData(GeographyFactory
-                .LineString(0.0, 0.0)
-                .LineTo(1.0, 0.0)
-                .LineTo(1.0, 1.0)
-                .LineTo(0.0, 1.0)
-                .LineTo(0.0, 0.0)),
+            new TestCaseData(new GeoPosition(1.0, 2.0)),
+            //new TestCaseData(new GeoLineString(new GeoPosition[]
```

Comment:
Nit: could just clean up the code to remove these since it's all in source control.